packages/app-store/_components/AppCard.tsx

```diff
@@ -29,20 +29,28 @@ export default function AppCard({
   const [animationRef] = useAutoAnimate<HTMLDivElement>();
 
   return (
-    <div className={`border-subtle mb-4 mt-2 rounded-md border ${!app.enabled && "grayscale"}`}>
-      <div className="p-4 text-sm sm:p-6">
+    <div
+      className={`border-subtle mb-4 ${app.isInstalled ? "mt-2" : "mt-6"} rounded-md border ${
+        !app.enabled && "grayscale"
+      }`}>
+      <div className={`${app.isInstalled ? "p-4 text-sm sm:p-4" : "px-5 py-4 text-sm sm:px-5"}`}>
```

Comment:
this should not be done with template literals and instead with classNames()

Reply:
will do that with classNames